
## Overview of pipelines offered by Genomics Platform

### DEMULTIPLEX

Demultiplexing pipeline converts raw images (.bcl) into .fastq.gz files according to samplesheet provided by the user. 
- Inputs: raw .bcl files and a sample sheet  
- Outputs: .fastq.gz files, multiQC report  
- Software used: bcl2fasta, multiQC  

![demux pipeline](/images/f01_demultiplex.pdf)
### RNAseq

RNAseq pipeline aligns reads in .fastq.gz to a specified reference genome.
- Inputs: .fastq.gz files, a reference genome  
- Outputs: indexed, sorted .bam files, read counts in .tsv format  
- Software used: STAR, multiQC  

![RNA pipeline](/images/f02_RNAseq.pdf)

### CHORseq

CHORseq pipeline aligns reads in .fastq.gz to a specified reference genome, using dm6 as spike-in.
- Inputs: .fastq.gz files, a reference genome  
- Outputs: indexed, sorted .bam files  
- Software used: bowtie2, samtools, picard, multiQC, custom scripts  

![CHOR pipeline](/images/f03_CHOR.pdf)

### ATACseq

ATACseq pipeline aligns reads in .fastq.gz to a specified reference genome (human GRCh38, mouse mm10, fly dmel-all-chromosome-r6.27). Blacklisted regions and reads mapped to mtDNA are also removed. BigWig files are generated by normalising to RPKM. 

- Inputs: .fastq.gz files, a reference genome
- Outputs: indexed, sorted .bam files, .bw files
- Software used: cutadapt, fastQC, bowtie2, samtools, picard, bedtools, bamCoverage

### CHIPseq

TBA

### COMMUNITY PIPELINES

Apart from maintaining in-house pipelines, Genomics Platform supports use of [snakePipes](https://snakepipes.readthedocs.io/en/latest/) and [nf-core](https://nf-co.re/) pipelines. There is a [dangpu-specific profile](https://github.com/nf-core/configs/blob/master/docs/ku_sund_dangpu.md) available for nf-core pipelines. 
